src/components/OSCALLoader.js

```diff
@@ -415,24 +358,20 @@ export function OSCALSSPLoader(props) {
       onResolutionComplete={onResolutionComplete}
       onFieldSave={(
         appendToLastFieldInPath,
-        data,
-        editedField,
+        partialRestData,
+        editedFieldJsonPath,
         newValue,
-        restMethod,
         restUrlPath
       ) => {
```

Comment:
The number of arguments here do not match up with the `props.onFieldSave` that is called in `OSCALEditableFieldActions`, which has support for 0 or 4 arguments.

Reply:
@mikeisen1, right, we're passing in a function that can take 5 arguments, but `OSCALEditableFieldActions` is only calling that function with 4 arguments, so `restUrlPath` will be null and `handleFieldSave` calls `restUtils.buildRequestUrl` which will construct the url from the `partialRestData` and `oscalObjectType`.

That works for simple field edits (as implied by the `OSCALEditableFieldActions` component name), and more complex operations like control editing would probably do better calling more specific REST util methods like `updateSspControlImplementationImplementedRequirementStatementByComponent` rather than trying to handle those scenarios with an overloaded `onFieldSave`, but again, I was trying to minimize the changes in this suggestion so did not yet wire that in to `OSCALControlPartEditor`